pages/advanced-algorithms/available-algorithms/pagerank_online.mdx

```diff
@@ -48,10 +48,18 @@ its expectation, which is Rank(v).
 | **Edge weights**    | unweighted |
 | **Parallelism**     | sequential |
 
+<Callout type="info">
+This algorithm is currently running in a sequential manner, but can be parallelized.
+If you have a demand for parallelizing this algorithm, please reach us over
+**[Discord](https://discord.gg/memgraph).**
+</Callout>
+
 ## Procedures
 
 Online PageRank should be used by executing the procedures in the following way:
-1. Parameters and the context is set using the `set()` procedure.
+1. With the `set()` procedure, the initial PageRank values are initialized on the nodes. This function
```

Comment:
What value is used for initialization? Random? Zero?

Reply:
sorry, I formulated it differently